core/src/main/java/cucumber/runner/Step.java

```diff
@@ -0,0 +1,89 @@
+package cucumber.runner;
+
+import cucumber.api.Pending;
+import cucumber.api.Result;
+import cucumber.api.Scenario;
+import cucumber.api.TestStep;
+import cucumber.api.event.TestStepFinished;
+import cucumber.api.event.TestStepStarted;
+import cucumber.runtime.AmbiguousStepDefinitionsException;
+import cucumber.runtime.DefinitionMatch;
+import cucumber.runtime.UndefinedStepDefinitionException;
+
+import java.util.Arrays;
+
+abstract class Step implements TestStep {
```

Comment:
I think `Step`should be reserved for gherkin steps (`gherkin.ast.Step`) and use `cucumber.runner.TestStep` (the same pattern as `cucumber.api.HookTestStep` and `cucumber.runner.HookTestStep`)

Reply:
Done.